install_config/install/prerequisites.adoc

```diff
@@ -592,9 +592,10 @@ Only required to be internally open on master hosts.
 
 | *1936*
 |TCP
-| For router statistics use. Required to be open when running the template
-router to access statistics, and can be open externally or internally to
-connections depending on if you want the statistics to be expressed publicly.
+| (*Optional*) Required to be open when running the template router to access
+statistics. Can be open externally or internally to connections depending on if
+you want the statistics to be expressed publicly. Can require extra
+configuration to open. See the Notes section below for more information.
```

Comment:
Would it makes sense to xref to notes section? (if it requires too much scrolling)

Reply:
It's barely a roll away, so I figured it was OK not to. It's on the same screen even.